Python/Python.sublime-syntax

```diff
@@ -1018,6 +1018,11 @@ contexts:
     - include: illegal-assignment-expression
     - match: ':'
       scope: punctuation.separator.annotation.variable.python
+      push:
+        - meta_scope: meta.variable.annotation.python
+        - match: (?=$|=|#|;)
+          pop: 1
+        - include: expression-in-a-group
```

Comment:
Could you introduce a named context for this? We generally try to avoid inline contexts since syntax definitions can inherit/extend others since it allows more finely tuning the exact contexts you need.

Reply:
Done